components/infobox/commons/infobox_person.lua

```diff
@@ -138,6 +139,7 @@ function Person:createInfobox()
 		Title{name = (args.informationType or 'Player') .. ' Information'},
 		Cell{name = 'Name', content = {args.name}},
 		Cell{name = 'Romanized Name', content = {args.romanized_name}},
+		Cell{name = 'Abbreviations', content = {args.abbreviations}},
```

Comment:
I don't think this is relevant for wikis other than F1, I'd suggest to put it in custom.
Otherwise move it to the alternate ID/nickname cells or make a customizable of these